test/spec/BpmnPropertiesPanelRenderer.spec.js

```diff
@@ -295,11 +289,7 @@ describe('<BpmnPropertiesPanelRenderer>', function() {
     it('should NOT bind with keyboard binding deactivated', async function() {
       const diagramXml = require('test/fixtures/simple.bpmn').default;
 
-      const { modeler } = await createModeler(diagramXml, {
-        keyboard: {
-          bind: false
```

Comment:
I think this is still relevant, i.e. `bind=false` is _still_ a valid configuration (and prevents keyboard binding).

Reply:
I'll revert it then

Reply:
what about the other test? I feel it is redundant now

Reply:
[Similar test](https://github.com/bpmn-io/diagram-js/commit/1978df96b29b6decf4a1010942cd490aed095e62#diff-87d5e39de1f93edcc7a6b562dbe634fa91ad7ea6dd248bf5f79cd3d1da420542L44-L75) was removed from `diagram-js`

Reply:
`bindTo` is old behavior, to be removed. `bind` is current (and serves a valid use-case).